frontend/src/pages/CycleCountItemCountPage.jsx

```
@@ -0,0 +1,135 @@
+      }
+    };
+    
+    fetchItem();
+    
+    // Cleanup
+    return () => controller.abort();
+  }, [id]);
+  
+  const handleSuccess = () => {
+    // Navigate back to the batch detail page
+    if (item && item.batch_id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)